bin/commands/usergroups_users.rb

```diff
@@ -0,0 +1,25 @@
+# This file was auto-generated by lib/tasks/web.rake
+
+desc 'Usergroups.users methods.'
```

Comment:
This looks strange. These should be subcommands of usergroups, see how it moved from the other file?